services/email_summary_service.py

```
@@ -143,10 +155,20 @@ def complete_processing_run(self, success: bool = True, error_message: Optional[
         # Finalize performance metrics
         self.performance_metrics['end_time'] = datetime.now()
         self.performance_metrics['total_emails'] = self.run_metrics['processed']
-        
+
+        # Send log to collector
+        status = "completed" if success else "failed"
+        log_level = "INFO" if success else "ERROR"
```

Ruff (0.13.3)

Local variable `log_level` is assigned to but never used

Remove assignment to unused variable `log_level`

(F841)